### DISCUSSION TITLE
Feature request: "wrap around" or allow adding the same screen twice (to work around special setups)

### DISCUSSION THREAD

**Discussion by maf-soft** [debauchee/barrier#1572] [Ideas]
I have 3 screens horizontally attached to my Windows machine, and now wanted to use the center one to work on my MacBook and use Windows just secondarily to the right and left for Browser / Email... After work I just want to use the 3 Windows screens again.

So I attached the Mac to this screen (it has 2 ports) and installed Barrier to share the Windows Mouse and Keyboard.

If I could create a server configuration like the following, it would work really well:
[windows] [mac] [windows]
(the 2 Windows screens have to be configured so that Windows thinks the left one is right and vice versa)

So the name of my server machine would appear twice, and I guess it can easily be implemented so you can leave the mac screen to the right AND left.

Alternative: add a "wrap around" checkbox, so you can leave ALL screens on both sides.

I guess this feature (or even both) would have some friends.

**Comment by Fr4cktail** [comment]
Did you find a way to do it? i have the exactly same scenario
